docs/quickstarts/README.md

```diff
@@ -40,21 +40,21 @@ You can use this simple [preview tool](https://quickstarts-content-preview.surge
 
 
 ## Summary of steps
-This is an overview of the steps you will need to complete to publish an interactive quick start or Learning resource tile in the Hybrid Cloud Console.
+This is an overview of the steps you will need to complete to publish an interactive quick start or Learning resource card in the Hybrid Cloud Console.
 
 1. Write your draft quick start content, and get review and approval from product stakeholders.
 
 2. When the content is ready, create your YAML quick start files in a new directory in the `docs` directory of [**Red Hat Insights quickstarts**](https://github.com/RedHatInsights/quickstarts/tree/main/docs/quickstarts), following the detailed instructions in this `README.md` file. 
 
-3. Get your quick start or Learning resource tile added to the [Hybrid Cloud Console](https://console.redhat.com):
+3. Get your quick start or Learning resource card added to the [Hybrid Cloud Console](https://console.redhat.com):
  
     a. Create a request in [the RHCLOUD Jira project](https://issues.redhat.com/projects/RHCLOUD/issues/) for engineering to merge the content. Add the `platform-experience-services` label to your Jira.
     
     b. Create a pull request (PR) to the `main` branch in the [**Red HatInsights/quickstarts**](https://github.com/RedHatInsights/quickstarts/) repository.
     
 4. The engineering team then merges the pull request to the Hybrid Cloud Console. The content will appear first on the [stage](https://console.stage.redhat.com/) of the Hybrid Cloud Console, and later in [production](https://console.redhat.com/).
```

Comment:
It would be helpful to know how long it will take to get your QS merged. It can take anywhere from 1 day to 3 weeks in my experience, so it would be good to get an estimation from the team.

Reply:
Good question - @Hyperkid123 can you advise on an estimate to provide here?

Reply:
It's tough to give an estimate, it depends on the services team and how quickly they can get to the PR. But as I have mentioned, I am open to adding a person from the docs team to the project with merge rights if that would help with the velocity.

Reply:
Since the timing is variable, I'm going to leave off this info for now - we can add it later as needed.